common/src/hooks/use-interactive-state.ts

```diff
@@ -1,6 +1,6 @@
 import { useCallback } from "react";
 import { useInteractiveState } from "@concord-consortium/lara-interactive-api";
-import { IInteractiveState } from "../../../types";
+import { IInteractiveState } from "../../../plant-growth/src/types";
```

Comment:
Shouldn't this be a `common` type rather than a sim-specific one?

Reply:
Yes, thanks, this import is totally wrong, probably an automatic VS Code import update while I was moving files around. I've just removed IInteractiveState type import in https://github.com/concord-consortium/alaskan-simulations/pull/42/commits/04529c394d913f00602586227fae28f5b055831a. Using `any` or any kind of generic interface doesn't chang much in this minimal hook (that might not need at all in fact, but let's keep it for now).